packages/addons/service/jellyfin/source/bin/jellyfin-downloader

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2021-present Team LibreELEC (https://libreelec.tv)
+
+. /etc/profile
+oe_setup_addon service.jellyfin
+
+ICON="$ADDON_DIR/resources/icon.png"
+CONTROL_FILE="/tmp/curl.done"
+DATA_FILE="/tmp/curl.data"
+JELLYFIN_FILE="jellyfin_@JELLYFIN_VERSION@.tar.gz"
+
+# check for enough free disk space
+if [ $(df . | awk 'END {print $4}') -lt 200000 ]; then
+  kodi-send --action="Notification(Not enough disk space, at least 200MB are required,30000,${ICON})" >/dev/null
+  exit 0;
+fi
+
+# remove install status and folders
+if [ -f $ADDON_DIR/extract.ok ]; then
+  rm $ADDON_DIR/extract.ok
+fi
+
+if [ -d $ADDON_DIR/libs ]; then
+  rm -rf $ADDON_DIR/libs
+fi
+
+if [ -d $ADDON_DIR/tmp_download ]; then
+  rm -rf $ADDON_DIR/tmp_download
+fi
+
+# create tmp download dir
+mkdir -p $ADDON_DIR/tmp_download
```

Comment:
Use `mktemp -d`? If everything goes to plan, it'll be deleted at the end so store it in tmpfs instead of writing to disk.

Reply:
Updated to use mktemp - 7403a4a925709ef977d1f7f0eebed08d273de9a3